### PR TITLE
SIG-windows-2022-annual-report

### LINKED ISSUE #7120: 2022 Annual Report: SIG Windows
Chairs: @marosset 
Liaison: @BenTheElder

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

A template has already been generated for your group, and is available [here](https://git.k8s.io/community/sig-windows/annual-report-2022.md).

Key dates:
- Initial PR to communtiy repo should be opened by March 24th, 2023
- PR should be reviewed and merged by April 7th, 2023

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-windows/annual-report-2022.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig windows
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by marosset**
**Which issue(s) this PR fixes**:

Fixes #7120

### CODE REVIEW

**Comment by marosset** [comment]
@jsturtevant @jayunit100 @aravindhp @claudiubelu 
This is ready for review. I'm sure I missed some stuff.

File: sig-windows/annual-report-2022.md [hunk 1]
```diff
@@ -24,86 +25,87 @@
...
-
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   Most of the SIG-Windows sub-projects do not have enough contributors
```
Comment (aravindhp) [inline sig-windows/annual-report-2022.md:33]: I would just say `The SIG-Windows sub-projects...` given that is case across all of them.

**Review by aravindhp — COMMENTED** [review]
Just a minor not. LGTM otherwise.

**Comment by marosset** [comment]
> Just a minor not. LGTM otherwise.

Thanks for taking a look.
I'll wait for a few more people to review and address feedback in bulk.

File: sig-windows/annual-report-2022.md [hunk 2]
```diff
@@ -4,17 +4,18 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - [HostProcess containers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md) graduated to `stable` in v1.26
+   - Windows performance tests are not reported on the [k8s perf dashboard](https://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019&metriccategoryname=E2E&metricname=CPUUsage)
```
Comment (jsturtevant) [inline sig-windows/annual-report-2022.md:8]:
```suggestion
   - Windows performance tests are now reported on the [k8s perf dashboard](https://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019&metriccategoryname=E2E&metricname=CPUUsage)
```

File: sig-windows/annual-report-2022.md [hunk 3]
```diff
@@ -4,17 +4,18 @@
 
 1. What work did the SIG do this year that should be highlighted?
```
Comment (jsturtevant) [inline sig-windows/annual-report-2022.md:5]: Should we add unit test improvements and the CRI KEP stuff?

Comment (jsturtevant) [reply sig-windows/annual-report-2022.md:5]: Oh I see the unit test stuff below

File: sig-windows/annual-report-2022.md [hunk 4]
```diff
@@ -24,86 +25,87 @@
     - [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support) - v1.26
     - [2802 - Identify Pod's OS during API Server admission](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission) - v1.25
 
-
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
```
Comment (jsturtevant) [inline sig-windows/annual-report-2022.md:30]: e2e triaging and fixing?

File: sig-windows/annual-report-2022.md [hunk 5]
```diff
@@ -24,86 +25,87 @@
     - [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support) - v1.26
     - [2802 - Identify Pod's OS during API Server admission](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission) - v1.25
```
Comment (claudiubelu) [inline sig-windows/annual-report-2022.md:27]: Does Node log viewer work count?

Also, we did start working on InPlace Pod Vertical Scaling support for Windows, but it did not merge yet (https://github.com/kubernetes/kubernetes/pull/112599)

File: sig-windows/annual-report-2022.md [hunk 6]
```diff
@@ -4,17 +4,18 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - [HostProcess containers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md) graduated to `stable` in v1.26
+   - Windows performance tests are not reported on the [k8s perf dashboard](https://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019&metriccategoryname=E2E&metricname=CPUUsage)
```
Comment (claudiubelu) [inline sig-windows/annual-report-2022.md:9]: Guide for adding Windows nodes using kubeadm with flannel and calico CNIs? https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/guides/guide-for-adding-windows-node.md

File: sig-windows/annual-report-2022.md (hunk 5 above)
Comment (claudiubelu) [reply sig-windows/annual-report-2022.md:27]: We also had the Windows pod sandbox stats information KEP: https://github.com/kubernetes/enhancements/pull/3439

File: sig-windows/annual-report-2022.md (hunk 3 above)
Comment (marosset) [reply sig-windows/annual-report-2022.md:5]: I'll add the CRI KEP

File: sig-windows/annual-report-2022.md (hunk 6 above)
Comment (claudiubelu) [reply sig-windows/annual-report-2022.md:9]: Prometheus Windows exporter for Kubernetes? https://github.com/prometheus-community/windows_exporter/pull/864

File: sig-windows/annual-report-2022.md (hunk 5 above)
Comment (marosset) [reply sig-windows/annual-report-2022.md:28]: node log viewer was 2023 :D
I added a note about the 2371 work

File: sig-windows/annual-report-2022.md (hunk 6 above)
Comment (marosset) [reply sig-windows/annual-report-2022.md:9]: Yes, let me add both of those!

Comment (marosset) [reply sig-windows/annual-report-2022.md:9]: Updated!

Comment (jsturtevant) [reply sig-windows/annual-report-2022.md:9]: are you missing the commit with these? I don't seem them...

Comment (marosset) [reply sig-windows/annual-report-2022.md:11]: Yes, i forgot to push :/
Should be there now!

**Pull request merged**